mocks/notification.go

```diff
@@ -0,0 +1,5 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: internal/notifications/notifications.go
+
+// Package mocks is a generated GoMock package.
+package mocks
```

Comment:
I don't think this file is meant to exist.

Reply:
Oh, might have been added by mistake